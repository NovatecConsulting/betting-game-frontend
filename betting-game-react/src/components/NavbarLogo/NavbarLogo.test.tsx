import React from "react";
import { shallow } from "enzyme";
import { NavbarLogo } from ".";

describe(NavbarLogo.name + " component", () => {
    it("should render and match snapshot.", () => {
        const wrapper = shallow(<NavbarLogo />);

        expect(wrapper.find("span")).toIncludeText("NOVATIPP");
        expect(wrapper).toMatchSnapshot();
    });
});
