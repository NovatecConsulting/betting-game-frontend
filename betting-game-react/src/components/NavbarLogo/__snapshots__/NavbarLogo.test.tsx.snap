// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavbarLogo component should render and match snapshot. 1`] = `
<div
  className="text-white mr-6 ml-10 inline"
  data-testid="NavbarLogoContainer"
>
  <span
    className="font-semibold text-xl tracking-tight"
    data-testid="NavbarLogo"
  >
    NOVATIPP
  </span>
</div>
`;
